lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -111,9 +111,9 @@ sub _before_render {
   return unless $has_spec or grep { $path =~ /^$_/ } @{$c->stash('openapi.base_paths')};
 
   my $format = $c->stash('format') || 'json';
-  my $res
+  my ($res, $status)
     = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : $c->openapi->not_implemented;
```

Comment:
I don't like list context. Please change to array ref, unless you have a good reason for returning a list.